internal/testhelpers/selfservice_recovery.go

```diff
@@ -43,7 +43,7 @@ func GetVerificationFlow(t *testing.T, client *http.Client, ts *httptest.Server)
 
 	rs, _, err := publicClient.V0alpha2Api.GetSelfServiceVerificationFlow(context.Background()).Id(res.Request.URL.Query().Get("flow")).Execute()
 	require.NoError(t, err, "%s", res.Request.URL.String())
-	assert.Empty(t, rs.Active)
+	assert.NotEmpty(t, rs.Active)
```

Comment:
As with the recovery strategy, these fields are no longer empty directly after creating a new flow, as we populate them with the currently active strategy.